pubspec.lock

```diff
@@ -49,7 +49,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.9.0"
+    version: "2.8.2"
```

Comment:
Please revert this

Reply:
done, 1 extra commit was created due to me having to pull from master cuz of conflicts